subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/ConfigurationInternal.java

```diff
@@ -37,6 +37,9 @@ enum InternalState {
         UNRESOLVED,
         BUILD_DEPENDENCIES_RESOLVED,
         GRAPH_RESOLVED,
+
+        // This state should be removed, but it is referenced by nebula gradle-resolution-rules-plugin.
```

Comment:
💭 Could it be deprecated?  It is internal.

Reply:
Done